.github/workflows/build_and_test.yml

```diff
@@ -192,14 +192,12 @@ jobs:
         # On windows use install + build so build directory is consistent across runs (i.e. don't build from conan cache)
         # Necessary because setting ccache base_dir doesn't currently work on windows
         run: |
-          conan install tket --user=tket --channel=stable -o boost/*:header_only=True -o with_all_tests=True -c tools.cmake.cmaketoolchain:generator=Ninja
           conan build tket --user=tket --channel=stable -o boost/*:header_only=True -o with_all_tests=True -c tools.cmake.cmaketoolchain:generator=Ninja
```

Comment:
Do we need `--build=missing` here? (and on line 202)

Reply:
No, we only need to add it when it was included in the preceding "install" command being removed.